benchmarks/upload_metrics_to_bq.py

```diff
@@ -238,8 +238,10 @@ def update_config_with_tuning_params(base_config: omegaconf.DictConfig,
 
 
 def main(argv: Sequence[str]) -> None:
-  # only write once
-  if int(os.environ["TPU_WORKER_ID"]) != 0:
+  is_pathways = (os.environ.get('JAX_PLATFORMS', '') == 'proxy')
+
+  # Only write once for McJAX. Pathways is single controller, so only can write once.
+  if not is_pathways or int(os.environ['TPU_WORKER_ID']) != 0:
     return
```

Comment:
Might be misunderstanding something here - don't we want this method to run for non-pathways users also?

Reply:
Thanks Sujeeth, yes that helps a lot